R/custom_functions.R

```diff
@@ -28,6 +28,10 @@ library(ComplexHeatmap)
 library(circlize)
 library(reshape2)
 library(ggtree)
+library(edgeR)
+library(tidyr)
+library(readr)
+library(inflection)
 
 #### Hack Seurat functions to plot geom_point (shape) and geom_text for label clusters, underlayed with a circles. This is key, as it identifies where the labels are plotted on the UMAP
```

Comment:
Rather than "hack" i would consider saying "modify" in the comment here.
Also, here and throughout, be sure to 'wrap' comment text so its within the page width

Reply:
Changed to "modify"